### ISSUE TITLE
[SF][A11y][List/Tree] With multi select enable and type inactive its not announcing disabled state 

### ISSUE THREAD

**Issue by saorabhkr** [SAP/ui5-webcomponents#6660]
### Describe the bug

Describe the bug
With List/Tree, when we have multi-select enabled and tree/list items are disabled/inactive, it's not getting announced by the keyword users. For disabled also it says selected. 

### Isolated Example

https://codesandbox.io/s/agitated-drake-fye15s?file=/src/App.js

### Reproduction steps

Go to [Demo Page](https://codesandbox.io/s/agitated-drake-fye15s?file=/src/App.js)
Give focus to the list 
See the announcement by NVDA/JAWS 
Instead of saying disabled its says - Selected 

### Expected Behaviour

It should announce the correct status,
- It should announce the right disabled staus the same way selected.

### UI5 Web Components for React Version

1.8.1

### UI5 Web Components Version

1.10.5

### Browser

Chrome

### Operating System

MacOS

### Additional Context

This is happening both for Tree and List

**Comment by plamenivanov91** [comment]
Hello @SAP/ui5-webcomponents-topic-p ,

The **disabled** state of a given **ListItem** is not announced with JAWS. Can we fix this?

Regards,
Plamen Ivanov

**Comment by dobrinyonkov** [comment]
Hi @saorabhkr,

The observed behavior that is consider incorrect is actually a side effect of a wrong configuration. The `ui5-list` component in a selection mode was not intended to display disabled items, as it goes against the Fiori guidelines and accessibility best practices, which state that items and their checkboxes should never be disabled. You can read more about this on https://experience.sap.com/fiori-design-web/list-overview.

When using the inactive type with a selection mode of the list, it should have no effect. Instead, if an action cannot be performed on the selected item(s), the user should be informed through an additional message or the list should not display them at all.

If you find this feature valuable, please provide us with your exact use case, and we will forward your feedback to our accessibility and UX experts.

Thanks and Best Regards,
Dobrin
